ResearchKit/Common/ORKFormItemCell.m

```diff
@@ -416,6 +416,25 @@ - (BOOL)textFieldShouldReturn:(UITextField *)textField {
     return YES;
 }
 
+- (BOOL)textField:(UITextField *)textField shouldChangeCharactersInRange:(NSRange)range replacementString:(NSString *)string {
+    ORKTextAnswerFormat *answerFormat = (ORKTextAnswerFormat *)[self.formItem impliedAnswerFormat];
+    
+    NSString *text = [textField.text stringByReplacingCharactersInRange:range withString:string];
+    text = [[text componentsSeparatedByCharactersInSet:[NSCharacterSet newlineCharacterSet]] componentsJoinedByString:@" "];
```

Comment:
Shouldn't it be joined with `""`? Since newline symbol are not counted in text length.